src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,63 @@
 package pl.javastart.task;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String serviceFile = "service.csv";
+        File file = new File(serviceFile);
+        Queue<Vehicle> vehicles = new LinkedList<>();
+        try {
+            vehicles = checkFileExist(serviceFile, file, vehicles);
+        } catch (FileNotFoundException e) {
+            System.out.println("Plik nie został utworzony");
+        }
+        boolean onOff = true;
+        while (onOff) {
+            System.out.println("""
+                    Wybierz co chcesz zrobić:
+                    0 - zakończyć
+                    1 - wczytać nowy pojazd do kolejki
+                    2 - obsłużyć pierwszy pojazd w kolejce""");
+            int userChoose = scanner.nextInt();
+            switch (userChoose) {
+                case 0:
+                    if (!vehicles.isEmpty()) {
```

Comment:
Jeżeli kolejka jest pusta, to też chcemy ją zapisać, żeby przy kolejnym uruchomieniu programu nie odczytywać pojazdów, które już obsłużyliśmy